docs/resources/tensorflow_frontend.md

```diff
@@ -13,8 +13,7 @@ Also, the frontend allows loading TensorFlow models in SavedModel, MetaGraph, an
 The current limitations:
```

Comment:
please replace `serialize` with `save_model` method in the diagram above

Reply:
I have replaced it in the https://github.com/openvinotoolkit/openvino/pull/18757/commits/93e39f6a8b70fe7cd1c7901d18029524011ff8d9 commit.